src/site/content/en/lighthouse-best-practices/no-document-write/index.md

```diff
@@ -1,47 +1,54 @@
 ---
 layout: post
-title: Page uses document.write()
+title: Uses document.write()
 description: |
-  Learn about `no-document-write` audit.
+  Learn how to speed up your page's load time by avoiding  document.write().
 web_lighthouse:
   - no-document-write
+updated: 2019-08-28
 ---
 
-For users on slow connections,
-such as 2G, 3G, or slow Wi-Fi,
-external scripts dynamically injected via `document.write()`
-can delay the display of main page content by tens of seconds.
-Lighthouse lists out every call to `document.write()`:
+Using [`document.write()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/write)
+can delay the display of page content by tens of seconds
+and is particularly problematic for users on slow connections.
+Chrome therefore blocks the execution of `document.write()` in many cases,
+meaning you can't rely on it.
+
+## How the Lighthouse `document.write()` audit fails
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags
+calls to `document.write()` that weren't blocked by Chrome:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="no-document-write.png" alt="Lighthouse audit showing usage of document.write">
-  <figcaption class="w-figcaption">
-    Page uses <code>document.write()</code>.
-  </figcaption>
+  <img class="w-screenshot" src="no-document-write.png" alt="Lighthouse audit showing usage of document.write">
 </figure>
 
-See [Intervening against `document.write()`](https://developers.google.com/web/updates/2016/08/removing-document-write)
-to learn more.
+For the most problematic uses,
+Chrome will either block calls to `document.write()`
+or emit a console warning about them, depending on the user's connection speed.
```

Comment:
```suggestion
or emit a Console warning about them, depending on the user's connection speed.
```

Capitalized because we're referring to the proper noun of the Chrome DevTools Console

Reply:
OK, since we use "DevTools Console" just a little later in the sentence, I'd propose this convention: "console" when we're talking about consoles in general (e.g., "console warning"), and "DevTools Console" when we're talking about Chrome. THoughts?

Reply:
As mentioned elsewhere I'm OK with whatever makes the most sense to you here